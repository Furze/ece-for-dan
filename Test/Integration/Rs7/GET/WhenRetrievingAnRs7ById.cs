//using MoE.Rolls.Domain.Event;

using Bard;
using MoE.ECE.Domain.Read.Model.Rs7;
using MoE.ECE.Integration.Tests.Chapter;
using MoE.ECE.Integration.Tests.Infrastructure;
using Shouldly;
using Xunit;
using Xunit.Abstractions;

namespace MoE.ECE.Integration.Tests.Rs7.GET
{
    public class WhenRetrievingAnRs7ById : SpeedyIntegrationTestBase
    {
        protected WhenRetrievingAnRs7ById(RunOnceBeforeAllTests testSetUp, ITestOutputHelper output,
            TestState<ECEStoryBook, ECEStoryData> testState) : base(testSetUp, output, testState)
        {
        }

        private const string Url = "api/rs7";

        protected override void Arrange()
        {
            Given
                .A_rs7_has_been_created()
                .An_rs7_is_ready_for_internal_ministry_review()
                .GetResult(storyData => Rs7Model = storyData.Rs7Model);
        }

        protected override void Act()
        {
            When.Get($"{Url}/{Rs7Model.Id.ToString()}");
        }

        private Rs7Model Rs7Model
        {
            get => TestData.Rs7Model;
            set => TestData.Rs7Model = value;
        }

        [Fact]
        public void ThenTheBusinessEntityIdShouldBePopulated()
        {
            Then
                .Response
                .Content<Rs7Model>()
                .BusinessEntityId
                .ShouldNotBeNull();
        }

        [Fact]
        public void ThenTheDeclarationShouldBePopulated()
        {
            var rs7Model = Then.Response
                .Content<Rs7Model>();

            rs7Model.Declaration.ShouldNotBeNull();

            rs7Model.Declaration?.FullName.ShouldNotBeEmpty();
            rs7Model.Declaration?.ContactPhone.ShouldNotBeEmpty();
            rs7Model.Declaration?.Role.ShouldNotBeEmpty();
            rs7Model.Declaration?.IsDeclaredTrue.ShouldNotBeNull();
        }

        [Fact]
        public void ThenTheOrganisationIdShouldBePopulated()
        {
            Then
                .Response
                .Content<Rs7Model>()
                .OrganisationId
                .ShouldNotBeNull();
        }

        [Fact]
        public void ThenTheResponseShouldBeOk()
        {
            Then
                .Response
                .ShouldBe
                .Ok<Rs7Model>();
        }

        [Fact]
        public void ThenTheRevisionShouldBePopulated()
        {
            var rs7AndRevision = Then.Response
                .Content<Rs7Model>();

            rs7AndRevision.RevisionNumber.ShouldNotBeNull();
            rs7AndRevision.RevisionDate.ShouldNotBeNull();

            rs7AndRevision.AdvanceMonths.ShouldNotBeNull();
            rs7AndRevision.EntitlementMonths.ShouldNotBeNull();
            rs7AndRevision.IsAttested.ShouldNotBeNull();
        }
    }
}