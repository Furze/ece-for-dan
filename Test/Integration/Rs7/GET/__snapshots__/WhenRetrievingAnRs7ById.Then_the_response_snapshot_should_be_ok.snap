{
  "Id": 177085,
  "BusinessEntityId": "3531cc3a-0403-479c-bd34-dc3f71e627b6",
  "OrganisationId": 114895,
  "FundingPeriod": "July",
  "FundingYear": 2021,
  "FundingPeriodYear": 2020,
  "IsZeroReturn": false,
  "RollStatus": "InternalReadyForReview",
  "ReceivedDate": null,
  "RevisionNumber": 1,
  "RevisionDate": "2020-11-12T14:59:00.7639705+13:00",
  "AdvanceMonths": [
    {
      "Id": 0,
      "MonthNumber": 7,
      "Year": 2020,
      "AllDay": 2,
      "Sessional": null,
      "ParentLed": null
    },
    {
      "Id": 0,
      "MonthNumber": 8,
      "Year": 2020,
      "AllDay": 2,
      "Sessional": null,
      "ParentLed": null
    },
    {
      "Id": 0,
      "MonthNumber": 9,
      "Year": 2020,
      "AllDay": 2,
      "Sessional": null,
      "ParentLed": null
    },
    {
      "Id": 0,
      "MonthNumber": 10,
      "Year": 2020,
      "AllDay": 2,
      "Sessional": null,
      "ParentLed": null
    }
  ],
  "EntitlementMonths": [
    {
      "Id": 0,
      "MonthNumber": 2,
      "Year": 2020,
      "Days": [
        {
          "DayNumber": 1,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 2,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 3,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 4,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 5,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 6,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 7,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 8,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 9,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 10,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 11,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 12,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 13,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 14,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 15,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 16,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 17,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 18,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 19,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 20,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 21,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 22,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 23,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 24,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 25,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 26,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 27,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 28,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 29,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        }
      ]
    },
    {
      "Id": 0,
      "MonthNumber": 3,
      "Year": 2020,
      "Days": [
        {
          "DayNumber": 1,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 2,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 3,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 4,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 5,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 6,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 7,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 8,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 9,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 10,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 11,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 12,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 13,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 14,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 15,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 16,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 17,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 18,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 19,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 20,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 21,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 22,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 23,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 24,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 25,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 26,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 27,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 28,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 29,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 30,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 31,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        }
      ]
    },
    {
      "Id": 0,
      "MonthNumber": 4,
      "Year": 2020,
      "Days": [
        {
          "DayNumber": 1,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 2,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 3,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 4,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 5,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 6,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 7,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 8,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 9,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 10,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 11,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 12,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 13,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 14,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 15,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 16,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 17,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 18,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 19,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 20,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 21,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 22,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 23,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 24,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 25,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 26,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 27,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 28,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 29,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 30,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        }
      ]
    },
    {
      "Id": 0,
      "MonthNumber": 5,
      "Year": 2020,
      "Days": [
        {
          "DayNumber": 1,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 2,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 3,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 4,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 5,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 6,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 7,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 8,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 9,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 10,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 11,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 12,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 13,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 14,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 15,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 16,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 17,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 18,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 19,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 20,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 21,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 22,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 23,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 24,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 25,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 26,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 27,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 28,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 29,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 30,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 31,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        }
      ]
    }
  ],
  "IsAttested": true,
  "Declaration": {
    "Id": 0,
    "FullName": "Donald Trump",
    "ContactPhone": "021509753",
    "Role": "President",
    "IsDeclaredTrue": true
  },
  "RequestId": "177085"
}
